kili/cli.py

```diff
@@ -21,17 +21,26 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 api_key_option = click.option(
-    '--api-key', type=str, envvar='KILI_API_KEY', required=True,
+    '--api-key', type=str, default=None,
     help=(
-        'Your Kili API key (overrides the KILI_API_KEY environment variable). '
-        'If not passed, requires the KILI_API_KEY environment variable to be set.'
+        'Your Kili API key (overrides the "KILI_API_KEY" environment variable). '
+        'If not passed, requires the "KILI_API_KEY" environment variable to be set.'
```

Comment:
rm "If not passed ..." since show_default is handling this:
```
--api-key TEXT     Your Kili API key (overrides the "KILI_API_KEY"
                     environment variable). If not passed, requires the
                     "KILI_API_KEY" environment variable to be set.  [default:
                     (current user ("KILI_API_KEY" environment variable))]
```